README.md

```diff
@@ -1,7 +1,18 @@
 # Backend
+
 Back end code repository for SENG499 Summer 2022 project.
 
+## Develop
+To run the backend locally you need to run a Postgres database and the GraphQL server separately. After cloning the repo, follow these steps to get things up and running:
+1. Run `npm ci` to install the project's dependencies
+1. Run `docker-compose up` to boot up a Docker container running the Postgres database
+2. Run `./start.sh` to start the GraphQL server at [`http://localhost:4000/`](localhost:4000). This script does the following:
```

Comment:
We should either rename `setup.sh` or change this to
```suggestion
2. Run `./setup.sh` to start the GraphQL server at [`http://localhost:4000/`](localhost:4000). This script does the following:
```
I think the formet is better since start.sh will be run every time you want to bring it up